test/unistd/links.c

```diff
@@ -41,11 +41,11 @@ void changedir(const char *dir) {
 }
 
 void setup() {
-  makedir("working");
+  makedir("/working");
```

Comment:
Can't we remove this now?

Reply:
Sadly, no, I wasn't able to covert this test to use `test/setup_nodefs.js`.

Right now, unlike other tests, this test relies on absolute paths, and `test/setup_nodefs.js` works by moving the current working directory to `/nodefs/`.